lib/ui/page/home/widget/block.dart

```diff
@@ -125,33 +125,42 @@ class Block extends StatelessWidget {
                 Container(
                   width: double.infinity,
                   padding: _aspected(context, padding),
-                  child: AnimatedSize(
-                    duration: const Duration(milliseconds: 300),
-                    alignment: Alignment.topCenter,
-                    curve: Curves.easeInOut,
-                    child: Column(
-                      crossAxisAlignment: crossAxisAlignment,
-                      mainAxisSize: MainAxisSize.min,
-                      children: [
-                        SizedBox(height: title != null ? 6 : 10),
-                        if (title != null) ...[
-                          Center(
-                            child: Container(
-                              padding: const EdgeInsets.fromLTRB(12, 0, 12, 6),
-                              child: Text(
-                                title!,
-                                textAlign: TextAlign.center,
-                                style:
-                                    titleStyle ??
-                                    style.fonts.big.regular.onBackground,
+                  child: ClipPath(
+                    clipper: _BottomEdgeClipper(),
+                    child: AnimatedSize(
```

Comment:
Won't this clipping applied to every single `Block` affect performance? https://docs.flutter.dev/release/breaking-changes/clip-behavior#summary